### DISCUSSION TITLE
Can I port the library for jetson board?

### DISCUSSION THREAD

**Discussion by rinechran** [microsoft/vcpkg#29803] [General]
https://github.com/pjueon/JetsonGPIO

I want to add a library to the vcpkg. But the question remains as to whether this is right.\

I wonder if it is right to add a library of a specific board to the vcpkg.

**Comment by JackBoosY** [comment]
You can add it with a specific version, but we cannot guarantee that it will not be updated in vcpkg.

**Reply by rinechran** [reply]
I can understand that. Thank you for your answer.
